prow/e2e-suite.sh

```diff
@@ -27,6 +27,8 @@ set -u
 set -x
 
 if [ "${CI:-}" == 'bootstrap' ]; then
+  export USER=Prow
```

Comment:
What will happen if `$CI` isn't `bootstrap`? Does it inherit `$USER` for some cases?

Reply:
These scripts are mainly for Prow, so the CI would be set to `bootstrap`. Otherwise, when people run these scripts on their own machine, we don't want to override the USER there.

Reply:
ah, I see. It's always set to be bootstrap on prow. Thanks for the info.